make/go/dep_jq.mk

```diff
@@ -9,28 +9,30 @@ $(call _assert_var,CACHE_VERSIONS)
 $(call _assert_var,CACHE_BIN)
 
 # Settable
-# https://stedolan.github.io/jq/download checked 20230201
-JQ_VERSION ?= 1.6
+# https://jqlang.github.io/jq/download 20230906 checked 20230907
+JQ_VERSION ?= 1.7
 
-# jq does not have an ARM release on Github so we'll use
-# the amd64 version via Rosetta
 ifeq ($(UNAME_OS),Darwin)
-JQ_OS := osx
-JQ_ARCH := -amd64
+JQ_OS := macos
+ifeq ($(UNAME_ARCH),x86_64)
+JQ_ARCH := amd64
+else
+JQ_ARCH := $(UNAME_ARCH)
```

Comment:
Should this just be `arm64` ?

Reply:
That's how it was before - I changed it to match this pattern in the protoc file: https://github.com/bufbuild/makego/blob/99dd87fd8e550f6875c3a83b18ec601db8c0aadc/make/go/dep_protoc.mk#L27C16-L27C29